### ISSUE TITLE
IOException (Server does not support byte ranges)

### ISSUE THREAD

**Issue by Willie30F** [chenxiaolong/Custota#29]
Custota App says:
> Failed to install OTA update
> IOException (Server does not support byte ranges)

When I test my server if it supports byte ranges using `curl -I https://p8pro-ota.someserver.de/husky-ota-uq1a.231205.015-c52b44d3.zip`. It outputs:
```
HTTP/1.1 200 OK
Server: nginx
Date: Thu, 21 Dec 2023 ***** GMT
Content-Type: application/zip
Content-Length: 2296984954
...
Accept-Ranges: bytes
```

Another test using `curl -I https://p8pro-ota.someserver.de/husky-ota-uq1a.231205.015-c52b44d3.zip -H "Range: bytes=0-1023"` It outputs:
```
HTTP/1.1 206 Partial Content
Server: nginx
Date: Thu, 21 Dec 2023 ***** GMT
Content-Type: application/zip
Content-Length: 1024
Last-Modified: Thu, 21 Dec 2023 ***** GMT
Connection: keep-alive
...
Content-Range: bytes 0-1023/2296984954
```

Seems to be supported. Any ideas?

**Comment by chenxiaolong** [comment]
Hmm, it looks like nginx only reports `Accept-Ranges: bytes` when `Range` wasn't specified in the request.

I'll adjust Custota's logic to check for `Content-Range` in the response instead.

**Comment by chenxiaolong** [comment]
This should be fixed by #30. I'll release version 2.5 with that fix in a little bit.

**Comment by Willie30F** [comment]
> This should be fixed by #30. I'll release version 2.5 with that fix in a little bit.

Thanks 🙏. I'll be happy to try it when it's released 👍.

**Comment by chenxiaolong** [comment]
Great, thanks for testing!
